modules/simple_forms_api/app/services/simple_forms_api/benefits_intake_client.rb

```diff
@@ -0,0 +1,192 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  class BenefitsIntakeClient
+    FORM_NUMBER_MAP = {
+      '20-10206' => 'vba_20_10206',
+      '20-10207' => 'vba_20_10207',
+      '21-0845' => 'vba_21_0845',
+      '21-0966' => 'vba_21_0966',
+      '21-0972' => 'vba_21_0972',
+      '21-10210' => 'vba_21_10210',
+      '21-4138' => 'vba_21_4138',
+      '21-4142' => 'vba_21_4142',
+      '21P-0847' => 'vba_21p_0847',
+      '26-4555' => 'vba_26_4555',
+      '40-0247' => 'vba_40_0247',
+      '40-10007' => 'vba_40_10007'
+    }.freeze
+
+    def initialize(current_user, form_number:, **kwargs)
+      @current_user = current_user
+      @form_number = form_number
```

Comment:
Do we ever use the `form_number` argument in initialization?

Reply:
Hmm, good point. I'm just assigning the global variable here, I could always do `@form_number = kwargs[:form_number]` instead.